_people-use-web/user-stories.md

```diff
@@ -19,9 +19,9 @@ support:
 {% include box.html type="start" h="2" title="Summary" class="full" %}
 {:/}
 
-The following stories are selected scenarios of people with disabilities using the Web, to highlight the effect of web accessibility barriers and the broader benefits of accessible websites and web tools.
+The following stories of people with disabilities using the Web highlight the effect of accessibility barriers and the broader benefits of accessible websites and web tools.
```

Comment:
+1 much more clear and concise.

Reply:
web should be lowercase but I’m happy to bother with that another time :-)

Reply:
Can be either according to our style guide :-) https://www.w3.org/WAI/EO/wiki/Style#Capitalization
* web - lowercase as adjective, such as "web accessibility", "web page", "web application"
* Web - can capitalize when referring to the World Wide Web, or leave lower case for consistency within a document